src/index.js

```diff
@@ -73,6 +73,14 @@ function initPolyfill() {
       "Error installing ScrollTimeline polyfill: could not attach WAAPI's getAnimations to document"
     );
   }
+
+  const origSupports = CSS.supports;
+  CSS.supports = (ident) => {
+    if (ident.includes('animation-timeline:')) {
+      ident = ident.replaceAll('animation-timeline', '--animation-timeline');
```

Comment:
We should ideally replace all polyfill supported properties, I think something like this:
```js
ident = ident.replaceAll(/(animation-timeline|scroll-timeline(-(name|axis))?|view-timeline(-(name|axis|inset))?|timeline-scope)\s*:/, '--supported-property:');
```

Also for encapsulation purposes we should only claim to support CSS properties if we've called initCSSPolyfill, i.e. this override of CSS.supports should be set up when initCSSPolyfill is called.

Reply:
Yes, replacing all properties was the goal here, ergo this still being a draft PR. Will update to the regex you provided.

Going to move the logic into `initCSSPolyfill` to make sure it only plugs CSS.supports() when actually necessary.